frontend/lib/graphql/equipment.graphql

```diff
@@ -82,6 +82,8 @@ fragment AttributeOverview on Attribute {
   ) {
     ...TimeSeriesValue
   }
+
+  description
```

Comment:
Looks like this field was already added to this fragment, so you can get rid of this repeat one